### PR TITLE
Will

### PR DESCRIPTION
**Pull request by LucyMoore**

### CODE REVIEW

File: client/routes.tsx [hunk 1]
```diff
@@ -2,9 +2,10 @@ import { Route, createRoutesFromElements } from 'react-router-dom'
 
 import AppLayout from './components/AppLayout.tsx'
 import Home from './components/Home.tsx'
+import SeedPlant from './components/SeedPlant.tsx'
 
 export const routes = createRoutesFromElements(
-  <Route path="/" element={<AppLayout />}>
```
Comment (michael-reisch) [inline client/routes.tsx:7]: change this back to app layout

**Review by michael-reisch — CHANGES_REQUESTED** [review]
Change routes back to original version

File: client/routes.tsx (hunk 1 above)
Comment (williford-abunas) [reply client/routes.tsx:7]: done

**Comment by LucyMoore** [comment]
confusion over where the pull requests are going -coming from so closing this one and Will will re send

**Comment by LucyMoore** [comment]
as above

**Review by michael-reisch — APPROVED** [review]

**Pull request merged**